<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Siilihai</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../siilihairesources.qrc">
    <normaloff>:/data/siilis_icon.png</normaloff>:/data/siilis_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QSplitter" name="forumsSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="hideButton">
                 <property name="toolTip">
                  <string>Show/Hide the group list</string>
                 </property>
                 <property name="text">
                  <string>Hide List</string>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextUnderIcon</enum>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>10</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="updateButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Update all foruns</string>
                 </property>
                 <property name="text">
                  <string>Refresh</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../siilihairesources.qrc">
                   <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextUnderIcon</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="stopButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Cancel updates</string>
                 </property>
                 <property name="text">
                  <string>Stop</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../siilihairesources.qrc">
                   <normaloff>:/data/process-stop.png</normaloff>:/data/process-stop.png</iconset>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextUnderIcon</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="viewInBrowser">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>View message in external browser</string>
                 </property>
                 <property name="text">
                  <string>Open</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../siilihairesources.qrc">
                   <normaloff>:/data/emblem-web.png</normaloff>:/data/emblem-web.png</iconset>
                 </property>
                 <property name="toolButtonStyle">
                  <enum>Qt::ToolButtonTextUnderIcon</enum>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="threadTree">
              <column>
               <property name="text">
                <string notr="true">Subject</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Date</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Author</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QScrollArea" name="scrollArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>798</width>
              <height>320</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="headerFrame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QFormLayout" name="formLayout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Date:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="messageDate">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Author:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="messageAuthor">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="messageSubject">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Subject:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWebView" name="webView">
               <property name="url">
                <url>
                 <string>about:blank</string>
                </url>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuForum">
    <property name="title">
     <string>Forum</string>
    </property>
    <addaction name="actionSubscribe_to"/>
    <addaction name="actionUnsubscribe"/>
    <addaction name="actionGroup_Subscriptions"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate_all"/>
    <addaction name="actionUpdate_selected"/>
    <addaction name="actionForce_update_on_selected"/>
    <addaction name="separator"/>
    <addaction name="actionReport_broken_or_working"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionParser_Maker"/>
   </widget>
   <widget class="QMenu" name="menuGroup">
    <property name="title">
     <string>Group</string>
    </property>
    <addaction name="actionMark_as_Read"/>
    <addaction name="actionMark_as_Unread"/>
   </widget>
   <widget class="QMenu" name="menuMessages">
    <property name="title">
     <string>Message</string>
    </property>
    <widget class="QMenu" name="menuView_as">
     <property name="title">
      <string>View as</string>
     </property>
     <addaction name="actionHTML"/>
     <addaction name="actionText"/>
     <addaction name="actionWeb_Page"/>
    </widget>
    <addaction name="menuView_as"/>
   </widget>
   <addaction name="menuForum"/>
   <addaction name="menuGroup"/>
   <addaction name="menuMessages"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSubscribe_to">
   <property name="text">
    <string>Subscribe to..</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
  <action name="actionGroup_Subscriptions">
   <property name="text">
    <string>Group Subscriptions..</string>
   </property>
  </action>
  <action name="actionUpdate_all">
   <property name="icon">
    <iconset resource="../../siilihairesources.qrc">
     <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update all</string>
   </property>
  </action>
  <action name="actionUpdate_selected">
   <property name="icon">
    <iconset resource="../../siilihairesources.qrc">
     <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update selected</string>
   </property>
  </action>
  <action name="actionUnsubscribe">
   <property name="text">
    <string>Unsubscribe..</string>
   </property>
  </action>
  <action name="actionMark_as_Read">
   <property name="text">
    <string>Mark as Read</string>
   </property>
  </action>
  <action name="actionMark_as_Unread">
   <property name="text">
    <string>Mark as Unread</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionWeb_Page">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Web Page</string>
   </property>
  </action>
  <action name="actionParser_Maker">
   <property name="text">
    <string>Parser Maker</string>
   </property>
  </action>
  <action name="actionReport_broken_or_working">
   <property name="text">
    <string>Report broken or working</string>
   </property>
  </action>
  <action name="actionForce_update_on_selected">
   <property name="icon">
    <iconset resource="../../siilihairesources.qrc">
     <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Force update on selected</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../siilihairesources.qrc"/>
 </resources>
 <connections/>
</ui>
